src/components/common/YourCompany/SummaryDefineCompany.vue

```diff
@@ -60,8 +60,8 @@ import { Component, Mixins, Prop } from 'vue-property-decorator'
 import { Getter } from 'vuex-class'
 
 // Components
-import { FolioInformation, BusinessContactInfo, OfficeAddresses } from '@/components/YourCompany'
-import { CorrectNameOptions } from '@/components/YourCompany/CompanyName'
+import { FolioInformation, BusinessContactInfo, OfficeAddresses } from '@/components/common/YourCompany/index'
```

Comment:
I don't think you need to specify `index`. I also saw this elsewhere (search for "/index") if you choose to clean this up.

Reply:
I see you fixed one of these, but I think there are more in the code.

Reply:
I suspect these are a result of directory renames, as i didn't/wouldn't put `/index` in the imports 😛 
Will have a search.

Reply:
Definitely, found a few more. Updated 👍